pkg/corerp/api/v20231001preview/environment_conversion_test.go

```diff
@@ -894,3 +987,160 @@ func Test_fromRecipeConfigEnvDatamodel(t *testing.T) {
 		})
 	}
 }
+
+func Test_getSecretsFromProviderProperties(t *testing.T) {
```

Comment:
The test name should be updated if the function name gets updated.

Reply:
updated